Tests/Functional/FailureMessagesTestCase/main.swift

```diff
@@ -121,6 +123,27 @@ class FailureMessagesTestCase: XCTestCase {
         XCTAssertGreaterThanOrEqual(-1, 0, "message", file: "test.swift")
     }
 
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertIdentical' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: test.swift:[[@LINE+7]]: error: FailureMessagesTestCase.testAssertIdentical : XCTAssertIdentical failed: \("a"\) is not identical to \("b"\) - message
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertIdentical' failed \(\d+\.\d+ seconds\)
+    func testAssertIdentical() {
+        let object = XCTestExpectation()
+        XCTAssertIdentical(object as AnyObject, object as AnyObject)
```

Comment:
Is it actually necessary to cast these using `as AnyObject` ? I would expect it to work without the casts

Reply:
With this specific assertion the cast wasn't needed indeed, I removed it.
All the other assertions require a cast to avoid a compiler error (`type of expression is ambiguous without more context`).